google-cloud-storage/samples/acceptance/acls_test.rb

```diff
@@ -112,7 +112,7 @@
     readers = bucket.file(remote_file_name).acl.readers
 
     out, _err = capture_io do
-      StoragePrintFileACL.new print_file_acl bucket_name: bucket.name,
```

Comment:
Minor: For readability, can we split this in two lines? One to instantiate the class and then the next to actually the method.